src/paging.md

```diff
@@ -352,10 +369,10 @@ another bit. This extra `1` is a ‘huge page’ bit, meaning that the pages are
 ```
 
 Just like before, we are now writing the value in `eax` to a location. But
-instead of it being just `p2_table + 0`, we’re adding `ecx * 8` Remember, `ecx`
-is our loop counter. Each entry is eight bits in size: `0b10000011`. So we need
-to multiply the counter by eight, and then add it to `p2_table`. Let’s take a
-closer look: let’s assume `p2_table` is zero, to make the math easier:
+instead of it being just `p2_table + 0`, we’re adding `ecx * 8`. Remember, `ecx`
+is our loop counter. Each entry is eight bytes in size, so we need to multiply
```

Comment:
Hm, I did think this was bites, not bytes. Do you have a citation somewhere?

I'm also coming back to this material after a long time away, so I may be mistaken.

Reply:
It can't be 8 bits, for at least a couple of reasons:
1. You're incrementing the address by 8 bytes each time. 8 bits is 1 byte. If each entry is only 1 byte, why are you incrementing by 8 bytes each time?
2. Each entry contains the address of a page. But we've used the bottom 8 bits for metadata. If the whole entry is 8 bits, and we've used 8 bits for metadata, where does the address go?

As for a citation, how about [Wikipedia](https://en.wikipedia.org/wiki/Physical_Address_Extension)? Or [OSDev](http://wiki.osdev.org/Setting_Up_Paging_With_PAE)?